examples/platforms/nrf528xx/src/radio.c

```diff
@@ -191,6 +204,86 @@ static inline void clearPendingEvents(void)
     } while (__STREXW(pendingEvents, (uint32_t *)&sPendingEvents));
 }
 
+#if OPENTHREAD_CONFIG_MAC_HEADER_IE_SUPPORT
+static void nrf_802154_tx_process_security(uint8_t *p_ack_frame)
```

Comment:
It seems there are very little common code between tx ack and tx other frames. It seems reasonable to separate code into corresponding functions: nrf_802154_tx_started() and nrf_802154_tx_ack_started(). Thoughts?